src/sentry/integrations/gitlab/integration.py

```diff
@@ -172,8 +172,10 @@ def search_projects(self, query):
         return client.search_projects(group_id, query)
 
     # TODO(cathy): define in issue ABC
-    def search_issues(self, project_id, query, iids):
+    def search_issues(self, query: str | None, **kwargs):
         client = self.get_client()
+        project_id = kwargs["project_id"]
+        iids = kwargs["iids"]
```

Comment:
Do we want guards here againt KeyErrors?

Reply:
i think we want these to behave like actual parameters so we should surface `KeyErrors` -- these are required in the client function